server/src/test/java/org/elasticsearch/common/settings/ScopedSettingsTests.java

```diff
@@ -1172,6 +1174,33 @@ public void testLoggingUpdates() {
         }
     }
 
+    public void testLoggingUpdateForRestrictedLogger() {
+        assumeFalse("Skipped, if TRACE_ENABLED restricted loggers are permitted", NetworkTraceFlag.TRACE_ENABLED);
```

Comment:
Looks like `es.insecure_network_trace_enabled` is enabled for `server` tests.
WDYT, is it worth it / acceptable to create an additional test task and have :server:test depend on it?